src/Parser/Priest/Discipline/Modules/Abilities.js

```diff
@@ -84,14 +84,6 @@ class Abilities extends CoreAbilities {
           suggestion: true,
         },
       },
-      {
```

Comment:
Do you have any information whether artifacts will actually be removed in the prepatch? We've been assuming they aren't just to be safe.

Reply:
Nothing concrete, but the fact that certain artifact abilities are becoming talents would point to even the passive abilities granted by the artifacts are going away.